src/app/schools/school.ts

```diff
@@ -1,24 +1,23 @@
 import {Entity} from '../entity/entity';
 
 export class School extends Entity {
-  static ENTITY_TYPE = 'School';
+  public static ENTITY_TYPE = 'School';
 
-  name = '';
-  medium = '';
-  privateSchool: boolean;
-  academicBoard = '';
+  public name = '';
```

Comment:
attributes are "public" by default. Any reason to add it here explicitly for some of the attributes?

Reply:
Your right, I am more used to do it that way but I will change it back to keep it in line with the project-style.

Reply:
Maybe also discuss it in the next meeting. If most people feel it's clearer if we explicitly add "public", we can also do that across all classes.